docs/eventing/samples/kafka/source/README.md

```diff
@@ -77,7 +77,14 @@ You must ensure that you meet the [prerequisites listed in the Apache Kafka over
 
 ### Create the Event Display service
 
-1. Build the Event Display Service (`even-display.yaml`)
+1. Download a copy of the code:
+
+   ```shell
+   git clone -b "{{< branch >}}" https://github.com/knative/docs knative-docs
+   cd knative-docs/docs/eventing/samples/kafka/source
+   ```
+
+2. Build the Event Display Service (`even-display.yaml`)
```

Comment:
```suggestion
2. Build the Event Display Service (`event-display.yaml`)
```

Reply:
I'm not sure that this is "building" the event-display program anyway.

Reply:
Ah, thanks, missed this, I'll make an update and fix it.